recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - setuptools
   run:
     - python
-    - smart_open >=5.0.0,<6.0.0
+    - smart_open >=5.2.1,<6.0.0
```

Comment:
Should be `smart-open >=5.2.1,<7.0.0`, see https://github.com/justindujardin/pathy/blob/b39b49206518f7e51490d69a240236c40261aadd/requirements.txt#L1

Reply:
thank you, addressed.